Tools/check-c-api-docs/main.py

```diff
@@ -0,0 +1,166 @@
+import re
+from pathlib import Path
+import sys
+import _colorize
+
+SIMPLE_FUNCTION_REGEX = re.compile(r"PyAPI_FUNC(.+) (\w+)\(")
+SIMPLE_MACRO_REGEX = re.compile(r"# *define *(\w+)(\(.+\))? ")
+SIMPLE_INLINE_REGEX = re.compile(r"static inline .+( |\n)(\w+)")
+SIMPLE_DATA_REGEX = re.compile(r"PyAPI_DATA\(.+\) (\w+)")
+
+MISTAKE = """\
+If this is a mistake and this script should not be failing, please create an
```

Comment:
No need for please in error messages:
```suggestion
If this is a mistake and this script should not be failing, create an
```